unit-test/config.json

```diff
@@ -3,7 +3,8 @@
   "spec_files": [
     "**/*[sS]pec.?(m)js",
     "**/*.js",
-    "!unit-test/script-overload-snapshots/out/*.js"
+    "!unit-test/script-overload-snapshots/out/*.js",
```

Comment:
Can you explain why we need this?

Reply:
'fixtures' is just a fake file that I wanted in place to test as much of the code-flow as possible - without this exclusion `jasmine` was trying to load it as a test file :)